### PR TITLE
[cxx-interop] Try more stdlib and overlay tests on Windows

### PR DESCRIPTION
**Pull request by hyp**

### CODE REVIEW

**Comment by hyp** [comment]
@swift-ci please test

**Comment by compnerd** [comment]
Please test with following PRs:
https://github.com/apple/swift/pull/67816

@swift-ci please test Windows platform

File: test/Interop/Cxx/stdlib/use-std-map.swift [hunk 1]
```diff
@@ -1,8 +1,4 @@
 // RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-experimental-cxx-interop)
-//
-// REQUIRES: executable_test
```
Comment (hyp) [inline test/Interop/Cxx/stdlib/use-std-map.swift:3]: oops, need to bring this back.

**Comment by compnerd** [comment]
Please test with following PRs:
https://github.com/apple/swift/pull/67816

@swift-ci please test Windows platform

**Comment by hyp** [comment]
@swift-ci please test and merge

**Pull request merged**